### PR TITLE
js-feat-navigation

### PR DESCRIPTION
**Pull request by sampoonachot**

### CODE REVIEW

File: client/styling/navigation.less [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.navigation {
+  float: right;
+
+  a {
```
Comment (rawfalafel) [inline client/styling/navigation.less]: I was hoping this would be an actual BEM component but this is a start :)

Comment (sampoonachot) [inline client/styling/navigation.less]: oh god is it not. ok hold up

Comment (sampoonachot) [inline client/styling/navigation.less]: @rawfalafel rdy

**Comment by rawfalafel** [comment]
there's a merge conflict 😭

File: client/app.jsx [hunk 2]
```diff
@@ -2,11 +2,11 @@ import React, { Component } from 'react'
 import Root from './components/root.jsx'
 
 class App extends Component {
-  render() {
-    return (
-      <Root />
-    );
-  }
+ render() {
```
Comment (rawfalafel) [inline client/app.jsx]: Looks like this diff only has one line?

**Pull request merged**